### PR TITLE
Add environment.yml and install.R to simplify installation

### PR DESCRIPTION
**Pull request by cookkate**
Ready to go. I took out the full list of R dependencies since I think it was already out of date. We'll have to keep on top of updating install.R as we add new ones.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,78 +50,28 @@ Therefore, it is a good idea to open the cloned repository in RStudio as a proje
...
-* viridis
-* vsn
-* vtree
-* yaml
-
-# Running the data analysis
+It is a good strategy to manage these particular software dependencies, required on your path, by setting up a Conda environment with them installed. For convenience, we have included an `environment.yml` file that can be used with conda to install these. By default this file creates a conda environment named 'R-ODAF'; you can [edit environment.yml](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-file-manually) to change this.
 
-## "Installing" the pipeline
+There are also numerous dependencies in R (a full list would take up a lot of space and would likely be incmoplete). Run `install.R` within R or using [Rscript](https://www.rdocumentation.org/packages/utils/versions/3.6.2/topics/Rscript) to install them.
```
Comment (mattjmeier) [inline README.md:67]: Typo in "incomplete"

**Review by mattjmeier — APPROVED** [review]
Looks great - I commented where I saw one typo

Will test this out as soon as I can!

**Comment by cookkate** [comment]
That prefix shouldn't be in the environment.yml file, I took it out. Will
figure out why that commit isn't being reflected here.

**Comment by mattjmeier** [comment]
No it actually isn't there anymore. I guess it only posted when I finished the review... I was attempting to comment that yesterday from mobile, but maybe it didn't actually post because I had accidentally started a review. I can see that it is red now and replaced.

**Pull request merged**